src/inclusion_proof.rs

```diff
@@ -25,7 +30,7 @@ pub fn generate_inclusion_proof(
     mut ext_headers: Vec<ExtHeaderRecord>,
     start_block: u64,
     end_block: u64,
-) -> Result<Vec<[H256; 15]>, EraValidateError> {
+) -> Result<Vec<[FixedBytes<32>; 15]>, EraValidateError> {
```

Comment:
Why the cahnge to `FixedBytes<32>`

Reply:
Trin changed to use AlloyPrimitives and it comes from that it seems, it is alright